NEWS.md

```diff
@@ -1,5 +1,17 @@
 ## 3.1.0 - Unreleased
 
+
+## 3.0.3 - Released (Quesnelia R1 2024)
+The primary focus of this release was updating edge-common dependency
```

Comment:
The primary focus of this release was updating edge-common dependency that
extends AwsParamStore implementation to support FIPS-approved crypto modules.

Reply:
Fixed